edge-apps/tfl-bus-status/README.md

```diff
@@ -0,0 +1,64 @@
+# TFL Bus Information Edge App
+
+This guide will walk you through the steps to create an TFL Bus Status Edge App in Screenly.
+
+
+## Prerequisites
+
+* Install Screenly CLI - Please follow the guide [here](https://github.com/Screenly/cli)
+
+## Steps
+
+Step 1. **Log in to the Screenly account via CLI**
+
+Follow the on-screen instructions to log in to your Screenly account.
+
+   `$ screenly login`
+
+Step 2. **Download and Open Edge App Playground**
+
+`$ git clone https://github.com/Screenly/Playground.git`
+
+Step 3. **Enter tfl-bus-status Edge App Directory**
+
+`$ cd edge-apps/tfl-bus-status`
+
+Step 4. **Create a New tfl-bus-status Edge App:**
+
+`$ screenly edge-app create --name TFL_EdgeApp --in-place`
+
+> Replace "TFL_EdgeApp" with your desired app name.
+
+Step 5. **Upload the Edge App**
+
+`$ screenly edge-app upload`
+
+> Wait for the upload to complete.
+
+Step 6. **Specify the TFL API and STOP ID** for example: 490005186S1.
+
+Replace "API" with the actual API that obtained from [https://api-portal.tfl.gov.uk/](https://api-portal.tfl.gov.uk/)
+
+`$ screenly edge-app secret set tfl_api=API`
+
+and provide the STOP ID also - replace the "stopID" with actual Stop ID.
```

Comment:
Where does one get the stop ID?

Reply:
Currently, there is an[ official CSV file](https://tfl.gov.uk/bus-stops.csv ) provided by the TFL team. But it's not user-friendly as the user needs to download the .csv file and needs to search for the stop name or stopID. I'm looking for a good alternative, or we might need to make a static page.

Reply:
Sorted in [here](https://playground.srly.io/edge-apps/helpers/tfl/bus-stop-lookup/) for now.

Reply:
Thanks @vpetersson , Will improve the UI later.